llama_index/llm_predictor/base.py

```diff
@@ -108,6 +138,7 @@ def _log_end(self, event_id: str, output: str, formatted_prompt: str) -> None:
             event_id=event_id,
         )
 
+    @_llm_predictor_retry
```

Comment:
Thoughts on adding this to the base LLM class instead? LLMPredictor itself isn't really user-facing anymore